library/src/main/kotlin/com/oddin/oddsfeedsdk/config/OddsFeedConfiguration.kt

```diff
@@ -34,11 +33,20 @@ class OddsFeedConfigurationBuilder internal constructor() {
     private var initialSnapshotRecoveryInterval: Duration? = null
 
     fun selectProduction() = apply {
-        selectedEnvironment = Environment.PRODUCTION
+        selectedEnvironment = Environment("mq.oddin.gg", "api-mq.oddin.gg", defaultMessagingPort)
     }
 
     fun selectIntegration() = apply {
-        selectedEnvironment = Environment.INTEGRATION
+        selectedEnvironment =
+            Environment("mq.integration.oddin.gg", "api-mq.integration.oddin.gg", defaultMessagingPort)
+    }
+
+    fun selectEnvironment(messagingHost: String, apiHost: String) = apply {
```

Comment:
Maybe name selectEnvironment is little bit confusing. Because here you do not select environment...  Best would be something setMqConfig, but this does not corespondent with selectProduction(). So maybe setEnvironmentConfig() ?